examples/tx-example.py

```diff
@@ -200,5 +215,14 @@ def get_userone_details():
 get_coin_info()
 get_account_assets()
 get_userone_details()
+# set timestamp to correct value
+# for more protobuf timestamp api info see:
+# https://googleapis.dev/python/protobuf/latest/google/protobuf/timestamp_pb2.html
+first_time=Timestamp()
+first_time.FromMilliseconds(1626249925185)
+last_time=Timestamp()
+last_time.FromMilliseconds(1626250078804)
```

Comment:
I don't think that the values should be hardcoded - will the code work about 1, 2, 3, ... years? I suggest to take current milliseconds in right time: https://stackoverflow.com/a/13200235/1350091